proto/claim/claim_record.proto

```diff
@@ -4,10 +4,14 @@ package tendermint.spn.claim;
 option go_package = "github.com/tendermint/spn/x/claim/types";
 
 import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
 
 message ClaimRecord {
-  string address   = 1;
-  string claimable = 2
-      [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false];
+  string address   = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  string claimable = 2 [
+    (gogoproto.nullable)   = false,
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
```

Comment:
Is `(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int"` still required?

Reply:
Yes the customtype annotations are still required.

I believe these will be used with upcoming changes and optimizations to protobuf comp.  As well as upcoming features like SIGN_MODE_TEXTUAL
- https://github.com/cosmos/cosmos-sdk/blob/0cee47dbd29d9809e48a858742e8b16ad5ea4990/docs/architecture/adr-050-sign-mode-textual.md
- https://github.com/cosmos/cosmos-sdk/blob/0cee47dbd29d9809e48a858742e8b16ad5ea4990/docs/architecture/adr-050-sign-mode-textual-annex1.md

Reply:
Ok I see, let's merge it in this PR 👍